pkg/status/status.go

```diff
@@ -0,0 +1,321 @@
+// Copyright 2023 Huawei Cloud Computing Technologies Co., Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package status
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/olekukonko/tablewriter"
+	"golang.org/x/crypto/ssh"
+	"openGemini-UP/pkg/config"
+	"openGemini-UP/pkg/exec"
+	"openGemini-UP/pkg/install"
+	"openGemini-UP/util"
+)
+
+const CheckProcessCommand = "ps aux | grep -E '(ts-meta|ts-sql|ts-store)' | grep -v grep | awk '{print $11}'"
+const CheckDiskCapicityCommand = "df -h | grep '^/dev/'"
+
+var AlwaysCheckPosrt = []int{8088, 8091, 8092, 8086, 8400, 8401, 8010, 8011}
```

Comment:
```suggestion
const CheckDiskCapacityCommand = "df -h | grep '^/dev/'"

var AlwaysCheckPorts = []int{8088, 8091, 8092, 8086, 8400, 8401, 8010, 8011}
```

AlwaysCheckPorts 这些端口固定 check 的吗，是不是应该读取拓扑文件配置的端口